IdSharp.Tagging/ID3v2/ID3v2Utils.cs

```diff
@@ -234,7 +234,7 @@ public static string ReadString(EncodingType textEncoding, byte[] bytes, int len
 
         public static string ReadString(EncodingType textEncoding, Stream stream, int length)
         {
-            string returnValue;
+            string returnValue = string.Empty;
```

Comment:
I think the intention was to have the compiler check returnValue was explicitly set on all code paths. I don't mind, this can stay. This method should be broken up into smaller methods anyway.